### PR TITLE
[6] add user sign up

### PR DESCRIPTION
**Pull request by MariuszBielecki288728**
Add sign up button
Add sign up form
Add sign up form validation
Add password hashing
After successful sign up, user is added to database and redirected to root path 
Email is unique

### CODE REVIEW

File: db/migrate/20181029020908_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+
```
Comment (kopciuch97) [inline db/migrate/20181029020908_create_users.rb]: empty line here

File: db/schema.rb [hunk 2]
```diff
@@ -58,4 +58,13 @@
...
 
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "password_digest"
+    t.index ["email"], name: "index_users_on_email", unique: true
+  end
+
```
Comment (kopciuch97) [inline db/schema.rb]: empty line here

Comment (MariuszBielecki288728) [reply db/schema.rb]: this is auto-generated file, so maybe I should leave this like that?

File: db/migrate/20181029020908_create_users.rb (hunk 1 above)
Comment (MariuszBielecki288728) [reply db/migrate/20181029020908_create_users.rb]: this is also auto-generated and in other files there are also empty lines before timestamps

**Pull request merged**